### PR TITLE
[Infra] update issue templates

### PR DESCRIPTION
**Pull request by KrzysztofMoch**
## Changes
  - issues templates - moved to beta templates that are grate improvement for users and maintainers
  - PR template - removed section about adding change log (we have automatic change log now)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: What platforms are you having the problem on?
+      multiple: true
+      options:
+        - iOS
+        - Android
+        - Windows
+        - visionOS
```
Comment (freeboub) [inline .github/ISSUE_TEMPLATE/bug-report.yml:31]:
```suggestion
        - VisionOS
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        - Android TV
+        - Apple tvOS
+  
+  - type: dropdown
```
Comment (freeboub) [inline .github/ISSUE_TEMPLATE/bug-report.yml:35]: maybe comment this until new architecture is supported ?

**Comment by freeboub** [comment]
few suggestions, but OK for me

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (KrzysztofMoch) [reply .github/ISSUE_TEMPLATE/bug-report.yml:35]: I think we should leave interop layer option

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (KrzysztofMoch) [reply .github/ISSUE_TEMPLATE/bug-report.yml:31]: Nah, it's should be small, like in iOS 😄

**Review by freeboub — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (KrzysztofMoch) [reply .github/ISSUE_TEMPLATE/bug-report.yml:31]: image
From apple docs

**Pull request merged**